### ISSUE TITLE
KITTI 2015 training will be black image

### ISSUE THREAD

**Issue by yokosyun** [JiaRenChang/PSMNet#200]
I'm using KITTI2015
model is stackhourglass
preraining is pretrained_model_KITTI2015.tar 

begining of training it produce good disparity map. but if I finetune the model with KITTI2015, the disparity map is getting worse

does any one have idea?

**Comment by yokosyun** [comment]
in main.py

KITTI dataset has minus disparity or zero disparity which is impossible

mask = (disp_true < args.maxdisp) 
mask = (disp_true > 0)  #you need to have this line!!!!!!! there is this line in finetuning.py so it will work
